http/json_schemas/message.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Message Schema",
+  "type": "object",
+  "properties": {
+    "headers": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "type": "object"
+        }
+      },
+      "required": ["properties"]
+    },
```

Comment:
In places when you have `Schema Object` type you should write reference to JSON Schema spec like:

```json
"query": {
  "$ref": "http://json-schema.org/draft-07/schema#"
}
```

Or use our AsyncAPI Schema -> https://github.com/asyncapi/asyncapi-node/blob/master/schemas/2.0.0.json#L359